src/CBT.NuGet.AggregatePackage/CBT/Module/After.Microsoft.Common.targets

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003" InitialTargets="NuGetRestoreGlobalPackages">
+
+  <PropertyGroup>
+    <RestoreNuGetPackagesDependsOn>$(RestoreNuGetPackagesDependsOn);AggregateNugetPackages</RestoreNuGetPackagesDependsOn>
+    <RestoreDependsOn>$(RestoreDependsOn);AggregateNugetPackages</RestoreDependsOn>
```

Comment:
This appears to be an issue in the globalpackages as well. 
..\..\src\CBT.NuGet.GlobalRestore\CBT\Module\After.Microsoft.Common.targets:    <RestoreDependsOn>RestoreGlobalNuGetPackages;$(RestoreDependsOn)</RestoreDependsOn>

I have filed an issue.